Bullwinkle.class.nut

```diff
@@ -385,7 +395,7 @@ class Bullwinkle {
 
             // Add the retry information
             message.retry <- {
-                "ts": time() + timeout,
+                "ts": ( Bullwinkle._isAgent() ? time() : hardware.micros()/1000000 )  + timeout,
```

Comment:
We could use ```hardware.millis()``` to would provide a much larger window (25 days), however, I chose ```hardware.micros()``` to provide some amount of consistency between the agent and device...

In my very ugly code, watchdog is still valid.  If time() is invalid and not ticking, the message fails before it ever gets to the watchdog.  If time() is ticking, the watchdog still works correctly by doing all of the nasty split() calls.